### ISSUE TITLE
Classification System should be improved

### ISSUE THREAD

**Issue by pablopb3** [fifaleaguedevs/fifaleaguedevs.github.io#63]
The classification should break the ties in the 2nd round of the league by particular golaverage. It should also comtemplate triple ties on points.

**Comment by charly3pins** [comment]
I totally agree with you. We have to improve the classification script because nowadays it's recalculating all matches each time we update it and it's not efficient and it doesn't take in consideration ties..
